src/pynwb/file.py

```diff
@@ -156,7 +157,7 @@ class NWBFile(MultiContainerInterface):
         {
             'attr': 'stimulus_template',
             'add': '_add_stimulus_template_internal',
-            'type': TimeSeries,
+            'type': (TimeSeries, Images),
```

Comment:
So you can add `Images` but not `Image` directly? I'm OK with that I just wanted to confirm that this is what we want.